scripts/update_ziggurat_tables

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env Rscript
+zig_cpp_tables <- function(template, dest) {
+  header <- "// Generated by scripts/update_ziggurat_tables - do not edit"
+
+  n <- 256
+  dat <- zig_constants(n)
+  r <- dat$r
+  v <- dat$v
+  x <- c(v / f(r), intervals(n, r, v))
+  y <- x[-1] / x[seq_len(n)]
+
+  format <- function(z) {
+    s <- vapply(z, deparse, "", control = "digits17")
+    paste(
```

Comment:
Do we know that this is enough precision? (is it based on being < 2.2e-16?)

Reply:
digits17 is the full precision of the underlying number